Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuth.h

```diff
@@ -140,6 +140,13 @@
 ///
 - (BOOL)clearStoredAccessTokens;
 
+///
+/// When NO, the Dropbox web login page will show a sign up link below the
```

Comment:
Can we elaborate on this a bit? In the past, some developers have been rejected on the App Store when the sign up link was showing, so I'd prefer if this was clearer as to why this option is here.

The HTTP docs document this with:

> When true users will not be able to sign up for a Dropbox account via the authorization page. Instead, the authorization page will show a link to the Dropbox iOS app in the App Store. This is only intended for use when necessary for compliance with App Store policies.